requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs
```

Comment:
That file already exists in `docs`.

Reply:
Resolved, the one in docs was ignored, so replaced it and removed the specified versions to use latest